app/plugins/eu.esdihumboldt.hale.app.transform/src/eu/esdihumboldt/hale/app/transform/ExecApplication.groovy

```diff
@@ -215,6 +221,30 @@ $baseCommand
 					warn('Unexpected parameter -providerId')
 				}
 				break
+
+			case '-filter':
+				executionContext.filters.addUnconditionalFilter(value);
+				break
+
+			case '-filter-on':
+			//<Type>:CQL:<EXPRESSION>
+				if(value.findIndexValues {it = ':'}.size() >=2 ){
+					int typeNameIndex = value.indexOf(':')
+					if(typeNameIndex>0){
+						String typeName = value.take(typeNameIndex)
+						String expression = value.substring(typeNameIndex+1)
+						executionContext.filters.addTypeFilter(typeName, expression)
+					}
+				}else {
```

Comment:
I thought you support using CQL as default for a filter? Then something like `River:width > 10` would be OK, or not?